search/searcher/search_geoboundingbox.go

```diff
@@ -157,68 +165,15 @@ func ComputeGeoRange(term uint64, shift uint,
 					return false
 				}
 				_ = reader.Close()
-			}
-			return true
+				return true
 		}
-	}
 
-	var computeGeoRange func(term uint64, shift uint) // declare for recursion
-
-	relateAndRecurse := func(start, end uint64, res, level uint) {
-		minLon := geo.MortonUnhashLon(start)
-		minLat := geo.MortonUnhashLat(start)
-		maxLon := geo.MortonUnhashLon(end)
-		maxLat := geo.MortonUnhashLat(end)
-
-		within := res%document.GeoPrecisionStep == 0 &&
-			geo.RectWithin(minLon, minLat, maxLon, maxLat,
-				sminLon, sminLat, smaxLon, smaxLat)
-		if within || (level == geoDetailLevel &&
-			geo.RectIntersects(minLon, minLat, maxLon, maxLat,
-				sminLon, sminLat, smaxLon, smaxLat)) {
-			codedTerm := makePrefixCoded(int64(start), res)
-			if isIndexed(codedTerm) {
-				if !within && checkBoundaries {
-					onBoundary = append(onBoundary, codedTerm)
-				} else {
-					notOnBoundary = append(notOnBoundary, codedTerm)
-				}
-			}
-		} else if level < geoDetailLevel &&
-			geo.RectIntersects(minLon, minLat, maxLon, maxLat,
-				sminLon, sminLat, smaxLon, smaxLat) {
-			computeGeoRange(start, res-1)
+	} else {
+		isIndexed = func([]byte) bool {
+			return true
 		}
 	}
-
-	computeGeoRange = func(term uint64, shift uint) {
-		if err != nil {
-			return
-		}
-
-		split := term | uint64(0x1)<<shift
-		var upperMax uint64
-		if shift < 63 {
-			upperMax = term | ((uint64(1) << (shift + 1)) - 1)
-		} else {
-			upperMax = 0xffffffffffffffff
-		}
-
-		lowerMax := split - 1
-
-		level := (GeoBitsShift1 - shift) >> 1
```

Comment:
Now that you aren't using this variable `GeoBitsShift1`, you could remove its declaration at line 27 in this file.

Reply:
That is still used elsewhere in the file (line 44)

Reply:
Ah ok - in that case an op can be saved below on line 251 within geoRangeCompute's relateAndRecurse(..) while initializing level ..
```
level := (GeoBitsShift1 - res) >> 1
```

Reply:
See if the most recent commit addresses what you meant.